app/presenters/return-requirements/add-note.presenter.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+/**
+ * Formats data for the `/return-requirements/{sessionId}/add-note` page
+ * @module AddNotePresenter
+ */
+
+function go (session) {
+  const data = {
+    id: session.id,
+    licenceRef: session.data.licence.licenceRef,
+    ..._transformSession(session.data)
+  }
+
+  return data
+}
+
+function _transformSession (sessionData) {
+  const currentNote = sessionData.note
+
+  if (!currentNote) {
+    return {
+      note: ''
+    }
+  }
+
+  return {
+    note: sessionData.note
+  }
+}
```

Comment:
Surely this won't work any more because of the changes made to how the note is stored in the session. It's now `{ note: { content: 'foo bar', userEmail: 'jo@example.com' }` ??

Plus, I think we can make it a bit simpler.

```suggestion
function go (session) {
  const { id, data } = session

  const pageData = {
    id,
    licenceRef: data.licence.licenceRef,
    note: data.note ? data.note.content : ''
  }

  return pageData
}
```

Reply:
This works because we return the object `{ content: 'foo bar', userEmail: 'jo@example.com' }` back to nunjucks template. We access it [here](https://github.com/DEFRA/water-abstraction-system/blob/f400bf60469e03b2df0d7b38833eac9f7cfeba41/app/views/return-requirements/check-your-answers.njk#L108)  using `note.content` and `note.userEmail` respectively. If you prefer `content` and `userEmail` to be sent as separate variables, we can do that instead.

I agree, using ternary operator to simplify the line makes does make it more succinct.

Reply:
amended the test and presenter. Rather than ternary, we could simplify line 14 to `data.note.content || ''` which yields the same result. I've added it to the PR, happy to use ternary if you object.